src/themes/custom/lazy-theme.module.ts

```diff
@@ -66,32 +70,22 @@ import { ForgotEmailComponent } from './app/forgot-password/forgot-password-emai
 import { ForgotPasswordFormComponent } from './app/forgot-password/forgot-password-form/forgot-password-form.component';
 import { ProfilePageComponent } from './app/profile-page/profile-page.component';
 import { RegisterEmailComponent } from './app/register-page/register-email/register-email.component';
+import { MyDSpacePageComponent } from './app/my-dspace-page/my-dspace-page.component';
 import { SubmissionEditComponent } from './app/submission/edit/submission-edit.component';
 import { SubmissionImportExternalComponent } from './app/submission/import-external/submission-import-external.component';
 import { SubmissionSubmitComponent } from './app/submission/submit/submission-submit.component';
-import { MyDSpacePageComponent } from './app/my-dspace-page/my-dspace-page.component';
-import { WorkflowItemSendBackComponent } from './app/workflowitems-edit-page/workflow-item-send-back/workflow-item-send-back.component';
 import { WorkflowItemDeleteComponent } from './app/workflowitems-edit-page/workflow-item-delete/workflow-item-delete.component';
-import { SubmissionModule } from '../../app/submission/submission.module';
-import { MyDSpacePageModule } from '../../app/my-dspace-page/my-dspace-page.module';
-import { NavbarComponent } from './app/navbar/navbar.component';
-import { HeaderComponent } from './app/header/header.component';
-import { FooterComponent } from './app/footer/footer.component';
+import { WorkflowItemSendBackComponent } from './app/workflowitems-edit-page/workflow-item-send-back/workflow-item-send-back.component';
 import { BreadcrumbsComponent } from './app/breadcrumbs/breadcrumbs.component';
-import { HeaderNavbarWrapperComponent } from './app/header-nav-wrapper/header-navbar-wrapper.component';
-import { FileSectionComponent } from './app/item-page/simple/field-components/file-section/file-section.component';
-import { SearchModule } from '../../app/shared/search/search.module';
-import { ResourcePoliciesModule } from '../../app/shared/resource-policies/resource-policies.module';
-import { ComcolModule } from '../../app/shared/comcol/comcol.module';
 import { FeedbackComponent } from './app/info/feedback/feedback.component';
 import { CommunityListComponent } from './app/community-list-page/community-list/community-list.component';
 
+// todo: optimize imports
```

Comment:
Should we create a separate ticket for this TODO task? Or is this sufficiently resolved? (Not sure what the TODO is reminding us to actually do)

Reply:
I think that's probably a note to myself I added when merging `main` & forgot to remove